zsh/common

```diff
@@ -21,7 +21,7 @@ fi
 vn() {
     if [[ -z "$1" ]]; then
         echo "Error: No argument provided"
-        return 1
+        return
```

Comment:
Prevents `cloud` from trying to send my phone to `rainbow` as a fallback if, for instance, $sprint is unset